front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -110,50 +130,68 @@ export default function AssistantNew({
       window.alert(`Error creating message: ${data.error.message}`);
       return;
     }
-
-    await router.push(`/w/${owner.sId}/assistant/${conversation.sId}`);
   };
 
   return (
     <AppLayout
       user={user}
       owner={owner}
+      isWideMode={conversation ? true : false}
       gaTrackingId={gaTrackingId}
       topNavigationCurrent="lab"
       subNavigation={subNavigationLab({ owner, current: "assistant" })}
+      titleChildren={
+        conversation && (
+          <ConversationTitle
+            title={conversation.title || ""}
+            shareLink={`${baseUrl}/w/${owner.sId}/assistant/${conversation.sId}`}
+            // onDelete={() => {}}
```

Comment:
nit

Reply:
we don't quite have a great story for that yet